docs/testrunner-toolkit.md

```diff
@@ -48,9 +48,9 @@ import TabItem from '@theme/TabItem';
 |8.3.0|Windows 10|Chrome, Firefox, MicrosoftEdge|
 |7.7.0|Windows 10|Chrome, Firefox, MicrosoftEdge|
 |7.3.0|Windows 10|Chrome, Firefox, MicrosoftEdge|
-|7.1.0|Windows 10|Chrome, Firefox, MicrosoftEdge|
-|6.6.0|Windows 10|Chrome, Firefox, MicrosoftEdge|
-|5.6.0|Windows 10|Chrome, Firefox, MicrosoftEdge|
+|7.1.0|Windows 10|Chrome, Firefox, MicrosoftEdge <small><span className="sauceGold">DEPRECATED</span></small> |
```

Comment:
Applies to all, shouldn't we specify a date of removal ?
So we're sure no-one is newly using it without proper warning ?